Sources/Extensions/MapKit/MKMapView+PovioKit.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  MKMapView+PovioKit.swift
+//  PovioKit
+//
+//  Created by Borut Tomažin on 11/11/2020.
+//  Copyright © 2020 Povio Labs. All rights reserved.
+//
+
+import MapKit.MKMapView
+
+public extension MKMapView {
+  /// Current zoom level
+  var zoom: Double {
```

Comment:
What does this denote, percent?

Reply:
Good question. This was taken out of context, therefore not good enough to be put here. I'll remove it.